src/commands/redis_cmd.cc

```diff
@@ -1614,34 +1614,39 @@ class CommandHGetAll : public Commander {
   }
 };
 
-class CommandHRange : public Commander {
+class CommandHRangeByLex : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
-    if (args.size() != 6 && args.size() != 4) {
-      return {Status::RedisParseErr, errWrongNumOfArguments};
+    CommandParser parser(args, 4);
+    while (parser.Good()) {
+      if (parser.EatEqICase("REV")) {
```

Comment:
why support `REV` option?

Reply:
This can be aligned with zrange and makes sense, for example if you need to take the largest data.

Reply:
About aligning zrange I will deal with it in the new pr.

Reply:
i think `hrangebylex` is enough currently since i heard there is such a demand. Developers may not be used to that `hash` support range operations, we can implement when someone wants it, WDYT?